### ISSUE TITLE
Some Strava activities are not synced.

### ISSUE THREAD

**Issue by torpeanders** [cpfair/tapiriik#214]
I sync Strava, Garmin Connect and Dropbox with Tapiriik. 

Yesterday I created a Smashrun account and authenticated with GC. After import I noticed it said my longest break is > 500 days.

Upon inspection I see that my workouts from the last months of 2012, all of 2013 and the first few months of 2014 are only present in Strava. I verified that they're missing in both GC and Dropbox.

**Comment by torpeanders** [comment]
This could very well be a user error. Noticed that neither Strava nor GC have been allowed to sync private activities. I've never noticed that option, and I've never knowingly created a private activity. Resyncing now, and will report back later.

**Comment by torpeanders** [comment]
Nope. That did not help. I tried to disconnect/connect to Strava, but that did not do it either. In "My Activities" I can see the missing activities listed, and I see that Tapiriik thinks they exist on all services.

**Comment by torpeanders** [comment]
Downloaded my bulk history from Strava now, and I see that I have 479 items there, but only 258 in Garmin Connect. Still Tapiriik says they are in sync.

**Comment by cpfair** [comment]
Do the missing activities appear in tapiriik.com/activities?

**Comment by torpeanders** [comment]
Yes. Created an issue on github after mailing you.
https://github.com/cpfair/tapiriik/issues/214

Do the missing activities appear in tapiriik.com/activities?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/cpfair/tapiriik/issues/214#issuecomment-206557877>

**Comment by torpeanders** [comment]
I see they have synced now. Thanks :) What was the issue?
